content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -49,34 +49,41 @@ meta:
 
 `.dvc` files can contain the following fields:
 
-- `outs` (always present): List of <abbr>output</abbr> entries that represent
-  the files or directories tracked with DVC. Typically there is only one per
-  `.dvc` file (but several can be added or combined manually).
-- `deps`: List of <abbr>dependency</abbr> entries for this stage, only present
-  when `dvc import` and `dvc import-url` are used. Typically there is only one
-  (but several can be added manually).
-- `wdir` (optional): Working directory for the stage command to run in (relative
-  to the file's location). If this field is not present explicitly, it defaults
-  to `.` (the file's location).
+- `outs` (always present): List of <abbr>output</abbr> entries (details below)
+  that represent the files or directories tracked with DVC. Typically there is
+  only one (but several can be added or combined manually).
```

Comment:
> Typically there is only one (but several can be added or combined manually).

Is this true for .dvc files? I think only one is present always... Need to check.

Reply:
Same in `deps` (line 57).

BTW, why not call the fields `out` and `dep` (singular) in that case?

Reply:
> Is this true for .dvc files? I think only one is present always... Need to check.

Yes.

> BTW, why not call the fields out and dep (singular) in that case?

Just for historic reasons. We were considering changing the format for those in 1.0 but decided not to. Probably will change in the future.

Reply:
Thanks for the answers Ruslan!

"Yes" this is true? This = "Typically there is only one (but several can be added or combined manually)."

From your other answer I think you mean Yes to This = "I think only one is present always..."

Sorry for the confusing question 😋

Reply:
@jorgeorpinel 

"Typically there is only one (but several can be added or combined manually)." is correct.